cmd/functions.go

```diff
@@ -75,6 +75,7 @@ var (
 	}
 
 	envFilePath string
+	unstable    = new(bool)
```

Comment:
Thank you for contributing. We probably don't want to expose this as a flag because we want maximum parity between serving locally and deploying to prod. And afaik, this is a runtime config and not part of deno bundle.

Reply:
I believe `deno bundle` does expose the unstable flag:
![Screenshot 2023-05-24 at 10 41 50 AM](https://github.com/supabase/cli/assets/1815885/ba29e9c0-cb8e-4f9b-9eba-51da6bd7abe8)

Reply:
> Thank you for contributing. We probably don't want to expose this as a flag because we want maximum parity between serving locally and deploying to prod. And afaik, this is a runtime config and not part of deno bundle.

I believe `deno bundle` does expose the unstable flag:
![Screenshot 2023-05-24 at 10 41 50 AM](https://github.com/supabase/cli/assets/1815885/ba29e9c0-cb8e-4f9b-9eba-51da6bd7abe8)